float64 acc_x_old
float64 acc_y_old
float64 acc_z_old
float64 acc_x_new
float64 acc_y_new
float64 acc_z_new

